.github/workflows/spellcheck.yml

```
@@ -0,0 +1,15 @@
+name: Check spelling
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: streetsidesoftware/cspell-action@v6
+        with:
+            files: "**/*.md"
+            config: .cspell.json
```

I would prefer a CSpell config not to be a dotfile by default.

```suggestion
            config: cspell.json
```

Dotfiles are often hidden in file managers by default, so such behaviour might not be handy enough for users who are unaware.

---

Sure, if you'd prefer it to be not hidden we can do that.